ui/next/app/app.tsx

```diff
@@ -59,10 +58,15 @@ import uiReducer from "./redux/ui";
 import metricsReducer from "./redux/metrics";
 
 import Layout from "./containers/layout";
-import { ClusterMain, ClusterTitle } from "./containers/cluster";
-import { DatabasesMain, DatabasesTitle } from "./containers/databases";
-import { HelpUsMain, HelpUsTitle } from "./containers/helpus";
-import { NodesMain, NodesTitle } from "./containers/nodes";
+import Cluster from "./containers/cluster";
+import ClusterOverview from "./containers/clusterOverview";
+import ClusterEvents from "./containers/clusterEvents";
+import Databases from "./containers/databases";
```

Comment:
That's my fault - I have a fix in a PR I'm cleaning up. If you need it to work ASAP just change [this line](https://github.com/cockroachdb/cockroach/blob/master/ui/next/app/containers/databases.tsx#L193) to `export default databasesMainConnected;`